### ISSUE TITLE
Certificate and CSRs are returned in PEM format, but checksums are for DER format

### ISSUE THREAD

**Issue by alekratz** [cloudflare/cfssl#792]
This is a minor thing, but I ended up having to trawl through the source code to figure out why my checksums weren't matching up. Could it be documented somewhere that the checksums are not calculated for the PEM data, but instead the raw certificate data?

**Comment by lziest** [comment]
what you mention "checksum", can you be more specific?

**Comment by alekratz** [comment]
Yes, sorry. When running the cfssl server with `cfssl serve`, the [API call for newcert](https://github.com/cloudflare/cfssl/blob/master/doc/api/endpoint_newcert.txt) returns a checksum of the certificate that was given back, both in MD5 and SHA1. However, it's not a checksum of the direct content returned; it's a checksum of the raw certificate data (i.e. in DER format). It feels counter-intuitive, and it feels like a "gotcha" that should probably be documented somewhere.

**Comment by Toilal** [comment]
Really confusing ...

**Comment by kisom** [comment]
@alekratz sorry, this was (is) motivated by internal requirements. I've updated the docs in #808.
